destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
The line spacing looks a little off here -- it seems like there's an extra indent.